NuRadioReco/detector/detector.py

```diff
@@ -151,6 +152,12 @@ def __init__(self, source='json', json_filename='ARIANNA/arianna_detector_db.jso
         """
         if(source == 'sql'):
             self.__db = buffer_db(in_memory=True)
+        elif source == 'dictionary':
+            self.__db = TinyDB(storage=MemoryStorage)
+            self.__db.purge()
+            stations_table = self.__db.table('stations', cache_size=1000)
+            for station in dictionary['stations'].values():
+                stations_table.insert(station)
```

Comment:
do I see it correctly that the implementation for channels is still missing?

Reply:
Oops, you are right. How did I miss this?